frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { NS, CLUSTER_STATUS } from '../utils/consts';
+import {
+  cancelAction,
+  confirmAction,
+  createStorageClassButton,
+  emptyStateBody,
+  storagePoolDropdown,
+  storagePool,
+} from '../views/multiple-pool';
+
+describe('Test ceph pool creation', () => {
+  
+  // Ceph status var
+  let cephStatus:any;
+
+  // Pool var
+  const poolName: string = 'example.com';
+  const replicaCount: string = '2';
+  const provisioner: string = 'openshift-storage.rbd.csi.ceph.com';
+
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    // cy.install();
+
+    // Check ceph cluster status
+    cy.exec(`oc get cephCluster -n ${NS} -o json`).then((res) => {
+      const cephValue = JSON.parse(res.stdout.toString());
+      cephStatus = cephValue.items[0];
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.clickNavLink(['Storage', 'Storage Classes']);
+
+    // View create storage class form
+    cy.byTestID(createStorageClassButton).click();
+  });
+
+  after(() => {
+    storagePool.delete(poolName);
+    cy.logout();
+  });
+
+  afterEach(() => {
+    checkErrors();
```

Comment:
```suggestion
    cy.checkErrors();
```
If you do it this way you won't have to add an import statement.

Reply:
But cy dont have any function with name checkErrors

Reply:
my bad it was previously a cypress command looks like it's no more.